RecoTracker/LSTCore/interface/Common.h

```diff
@@ -57,6 +57,12 @@ namespace lst {
   };
 #endif
 
+
+  struct Params_Embedding {
+    // Embedding is a fixed array of 6 floats.
+    using ArrayFx6 = edm::StdArray<float, 6>;
+  };
```

Comment:
from the implementation detail it looks like this should be a part of `Params_T5`